benchmark/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM nvidia/cuda:12.2.0-runtime-ubuntu22.04
+
+RUN ln -snf /usr/share/zoneinfo/Europe/Zurich /etc/localtime && \
+    echo Europe/Zurich > /etc/timezone && \
```

Comment:
Why is setting the time zone necessary?

Reply:
It is not.
But in some cases, setting it avoids annoying prints of unset locales.

Reply:
I still think it's better to remove it since specifying the time zone does not seem necessary. But keep it if you prefer.